src/queries/eventRecords.js

```diff
@@ -1,8 +1,8 @@
 import gql from 'graphql-tag';
 
 export const GET_EVENT_RECORDS = gql`
-  query EventRecords($limit: Int, $order: EventRecordsOrder) {
-    eventRecords(limit: $limit, order: $order) {
+  query EventRecords($limit: Int, $offset: Int, $order: EventRecordsOrder) {
+    eventRecords(limit: $limit, skip: $offset, order: $order) {
```

Comment:
offset = asking for the next two in the list?

Reply:
with `offset`, the query returns not from the first element, but from the element index given per `offset` query variable.